modules/programs/vivid.nix

```diff
@@ -0,0 +1,141 @@
+{ config, lib, pkgs, ... }:
+let
+  yaml = pkgs.formats.yaml { };
+  bashLine = theme:
+    ''export LS_COLORS="$(${lib.getExe pkgs.vivid} generate ${theme})"'';
+  fishLine = theme:
+    "set -gx LS_COLORS (${lib.getExe pkgs.vivid} generate ${theme})";
+  nushellLine = theme: "${lib.getExe pkgs.vivid} generate ${theme}";
+  zshLine = bashLine;
+in {
```

Comment:
You can put a `with lib;` declaration here, so usages of `lib` afterward can be shorten.